nasa-gibs/worldview
# WV-2939 EIC carbon

## Description
At EIC, the Carbon Monoxide imagery on Worldview is reverting to October 6th even though there is more recent imagery available. This is because there has been no imagery for about 2 weeks & the EIC tile check function only checks the most recent 7 days before applying a known good fallback date. This problem is fixed by updating the fallback dates for each layer used in the EIC. You can confirm that this layer reverts to October 6 using [this link](https://worldview.earthdata.nasa.gov/?v=-203.47763385925995,-141.2590113285273,203.47763385925995,146.2590113285273&df=true&kiosk=true&eic=si&l=Coastlines_15m,AIRS_L3_Carbon_Monoxide_500hPa_Volume_Mixing_Ratio_Daily_Night,AIRS_L3_Carbon_Monoxide_500hPa_Volume_Mixing_Ratio_Daily_Day&lg=false).

Fixes # wv-2939

## How To Test

- git checkout wv-2939-eic-carbon
- npm run watch
- Load the same link as above on your [local instance](http://localhost:3000/?v=-203.47763385925995,-141.2590113285273,203.47763385925995,146.2590113285273&df=true&kiosk=true&eic=si&l=Coastlines_15m,AIRS_L3_Carbon_Monoxide_500hPa_Volume_Mixing_Ratio_Daily_Night,AIRS_L3_Carbon_Monoxide_500hPa_Volume_Mixing_Ratio_Daily_Day&lg=false).
- Confirm that the dates reverts to October 17th instead of October 6th.
- Review the notes in the ticket & check that the other EIC panels are working correctly.

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
